test/core/card.spec.js

```diff
@@ -3,20 +3,15 @@ const expect = require('./../helper').expect;
 const Card = require('./../../src/core/card');
 
 describe('Card', () => {
-  beforeEach(() => {
-    this.card = new Card(2, 'spades');
-  });
+  const cardData = [2, 'spades'];
+  const card = new Card(...cardData);
```

Comment:
You're still instantiating the card once for all the tests. All the code must be always in a block (`it`, `beforeEach`, ...)